release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1158,6 +1220,31 @@ In the following tables, features are marked with the following statuses:
 
 |====
 
+[discrete]
+=== OpenShift CLI (oc) Technology Preview features
+
+.OpenShift CLI (oc) Technology Preview tracker
```

Comment:
All the three key features were introduced through oc-mirror v2 and they are tech preview in 4.16, So that is the reason I have added 'Not Available' for 4.14 and 4.15. Should I remove them?

Reply:
Yeah, maybe the oc-mirror Plugin v2 entry is enough as this groups features.